components/stories/StoryDetail.tsx

```diff
@@ -8,9 +8,12 @@ interface Props {
   onClose: () => void
 }
 
+// TODO: This is gross, how can this be done better?
+const minHeight = 2000
+
```

Comment:
nit: Does this need to be a `const` - can it just be passed in as a prop where it's used?